modules/Model.js

```diff
@@ -1,23 +1,58 @@
 (function(root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define(['underscore', 'backbone', './pollingMixin'], factory);
+    define(['underscore', 'backbone', './pollingMixin', './Logger'], factory);
   } else if (typeof exports === 'object') {
-    module.exports = factory(require('underscore'), require('backbone'), require('./pollingMixin'));
+    module.exports = factory(require('underscore'), require('backbone'), require('./pollingMixin'), require('./Logger'));
   } else {
     root.Torso = root.Torso || {};
-    root.Torso.Model = factory(root._, root.Backbone, root.Torso.Mixins.polling);
+    root.Torso.Model = factory(root._, root.Backbone, root.Torso.Mixins.polling, root.Torso.Logger);
   }
-}(this, function(_, Backbone, pollingMixin) {
+}(this, function(_, Backbone, pollingMixin, Logger) {
   'use strict';
 
-  /**
-   * Generic Model
-   * @module    Torso
-   * @class     Model
-   * @constructor
-   * @author kent.willis@vecna.com
-   */
-  var Model = Backbone.Model.extend({});
+
+  var Model = Backbone.Model.extend({
+
+    wrapError: function(options){
+      var error = options.error;
+      options.error = function(resp) {
+        if (error) error(model, resp, options);
+        model.trigger('error', model, resp, options);
+      };
+    },
+
+    /** 
+    * Overridden to send before and after signals to Logger
+    */
+    fetch: function(options){
+      var UUID = "uuid-"+(new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+      Logger.track({
+        UUID : UUID,
+        type : "fetch",
+        state: "start",
+        time: Date.now(),
+      });
+
+      options = options ? _.clone(options) : {};
+      if (options.parse === void 0) options.parse = true;
```

Comment:
did you have to copy the original fetch method? We want to prevent this if at all possible
Can you instead call the parent's prototype and wrap that with what you want?